assets/dev/js/frontend/utils/flex-horizontal-scroll.js

```diff
@@ -70,7 +70,7 @@ function getChildrenWidth( children ) {
 }
 
 function initialScrollPosition( element, direction, justifyCSSVariable ) {
-	const isRTL = elementorCommon.config.isRTL;
+	const isRTL = elementorFrontend.config.isRTL;
```

Comment:
```suggestion
	const isRTL = elementorFrontend.config.is_rtl;
```
Did you mean to use this? I don't see `isRTL` 🤔  If I am correct, there is a reference to `isRTL` in `base-carousel.js` that also needs to change.

Reply:
Updated.
Inside the `base-carousel.js` as well.

Reply:
Thanks @hein-obox . How reliable is the E2E tests if it did not pick this up? 🤔 (I mean all tests passed, but the fix could not have worked).

Reply:
In this case, my test doesn't test the actual scenario. Previously we only had editor tests, but we didn't have any frontend tests. I have added a front-end test.

But what I haven't added, is a test inside incognito mode. I think it will take a while to write such a test.
I thought that might be too far-fetched.
What do you think?
@louiswol94

Reply:
Replied via DM.

Reply:
@louiswol94 Updated.